docs/xamarin/upgrade-23x-android.mdx

```diff
@@ -40,21 +40,21 @@ The Helpshift SDK v2.4.0 is a major update. Please note the we have deprecated s
 
 On SDK version 2.3.1 or below, you can check if an active conversation exists as shown below:
 
-```
+```java
```

Comment:
csharp. In fact, all code in this file is csharp, not java

Reply:
Ack